website/content/api-docs/volumes.mdx

```diff
@@ -275,10 +294,10 @@ $ curl \
 }
 ```
 
-## Register Volume
+## Register CSI Volume
```

Comment:
just noting that this and the other CSI headers will change the anchor link, which could break folks' bookmarks

`/volumes#register-volume` => `/volumes#register-csi-volume`

I do think it's an improvement, but wanted to mention the trade-off.

Reply:
Yeah, I don't think there's a good way around that unfortunately without leaving vague section headers.